packages/grid/x-data-grid/src/components/panel/filterPanel/GridFilterInputMultipleSingleSelect.tsx

```diff
@@ -93,7 +83,7 @@ function GridFilterInputMultipleSingleSelect(props: GridFilterInputMultipleSingl
       multiple
       freeSolo={false}
       limitTags={1}
-      options={resolvedValueOptions}
+      options={resolvedValueOptions as any} // TODO: avoid `any`?
```

Comment:
Feel free to suggest better solution here

Reply:
```tsx
    <Autocomplete<ValueOptions, boolean, boolean, boolean>
```

I tried doing that
But it's a rabbit-hole to type correctly all the props... I propose to keep the `any`